src/Cmfd.cpp

```diff
@@ -418,26 +395,26 @@ void Cmfd::computeDs(int moc_iteration) {
           else{
 
             /* Set properties for cell next to Surface */
-            if (surface == 0) {
+            if (surface == SURFACE_X_MIN) {
```

Comment:
Same as above; also, if `surface` is equal to `SURFACE_X_MIN` or `SURFACE_X_MAX`, `next_length_perpen` will be set to the `_cell_width` variable.